dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task210527DropReviewFieldsFromContentletTable.java

```diff
@@ -0,0 +1,36 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+import com.dotmarketing.util.Logger;
+import java.sql.SQLException;
+
+public class Task210527DropReviewFieldsFromContentletTable implements StartupTask {
+    @Override
+    public boolean forceRun() {
+        try {
+            final DotDatabaseMetaData dbMetadata = new DotDatabaseMetaData();
+            return dbMetadata.hasColumn("contentlet", "last_review") &&
+                    dbMetadata.hasColumn("contentlet", "next_review") &&
+                    dbMetadata.hasColumn("contentlet", "review_interval");
+        } catch (SQLException e) {
+            Logger.error(this, e.getMessage(),e);
+            return false;
+        }
+    }
+
+    @Override
+    public void executeUpgrade() throws DotDataException, DotRuntimeException {
+        try {
+            final DotConnect dotConnect = new DotConnect();
```

Comment:
don't you need to set the autoCommit to true?

Reply:
not needed. It ran ok in all databases

Reply:
I would do the same Question, msSQL?